docarray/array/storage/base/backend.py

```diff
@@ -75,7 +75,10 @@ def _map_id(self, _id: str) -> str:
         return _id
 
     def _map_column(self, value, col_type) -> str:
-        return self.TYPE_MAP[col_type].converter(value)
+        if value is not None:
+            return self.TYPE_MAP[col_type].converter(value)
+        else:
+            raise ValueError('Column values should not be None.')
```

Comment:
there's no other backend that supports nullable fields ?

Reply:
Actually other backend also don't support nullbale fields, like None for n_dim in other backends, I think it is better to keep consistent for all the backends.

Reply:
we have to aim to enable `nullable` fields wherever we can

Reply:
yes I agree I would say allow `nullable` should be the default and we deactivate it for some given backend that don't support it